ui/app/AppLayouts/Wallet/services/dapps/WalletConnectService.qml

```diff
@@ -28,6 +28,7 @@ QObject {
 
     readonly property alias dappsModel: dappsProvider.dappsModel
     readonly property alias requestHandler: requestHandler
+    readonly property int loginType: root.store.loginType
```

Comment:
I'm not sure bringing account state to service is a good idea. This is embedded with the account, right? Can't it be extracted from the account?

Reply:
On a second thought `selectedAccountAddress` also looks like an unnecessary state added tot he service interface, why not extract it from the `walletRootStore` directly?

Reply:
Ohh, you're right! Thanks! 🍻 
Was looking at this service as the only dependency it needs to communicate with the outside and maybe extract it someday out of the UI component. But IDK I've totally dismissed the idea of adding another input properties in the `DAppsWorkflow`.

Fixing it :D

Reply:
Done! Please have a look here https://github.com/status-im/status-desktop/pull/15520/commits/84053e2b2abaee3fca748eca48519404be2745ad

Sign transaction is fixed as well.

Thanks for this! 👍